lix/source-code/fs/src/NodeishFilesystemApi.ts

```diff
@@ -26,6 +26,7 @@ export type NodeishFilesystem = {
 		options?: {
 			recursive?: boolean
 			signal?: AbortSignal
+			persistent?: false
```

Comment:
this is not nodeish API spec. the spec is default persistent to true right? If so, please remove this.

Reply:
@samuelstroschein  we only support persistent: false .  this is supposed to not be the default value but the only valid value, happy for a better type proposal.

Reply:
If we remove this, the builds are stuck. @samuelstroschein.